src/jit/inline.cpp

```diff
@@ -1456,10 +1467,20 @@ void InlineStrategy::DumpXml(FILE* file, unsigned indent)
         fprintf(file, "<InlineForest>\n");
         fprintf(file, "<Policy>%s</Policy>\n", m_LastSuccessfulPolicy->GetName());
 
-        if (JitConfig.JitInlineDumpData() != 0)
+        const int dumpDataSetting = JitConfig.JitInlineDumpData();
+        if (dumpDataSetting != 0)
         {
             fprintf(file, "<DataSchema>");
-            DumpDataSchema(file);
+
+            if (dumpDataSetting == 1)
```

Comment:
I think that minimally this code should also have the descriptions of what "1" and "2" mean, or preferably that you should go ahead and define an enum.